content/docs/deployment/csminstallationwizard/src/static/js/tests/generate-yaml.test.js

```diff
@@ -490,18 +490,18 @@ describe("GIVEN createYamlString function", () => {
           enabled: $HEALTH_MONITOR_ENABLED
         nodeSelector: $NODE_POD_NODE_SELECTOR
       csireverseproxy:
-        image: dellemc/csipowermax-reverseproxy:v2.5.0
+        image: quay.io/dell/container-storage-modules/csipowermax-reverseproxy:v2.5.0
```

Comment:
Are we using the latest images version? csipowermax-reverseproxy:v2.5.0 it should be csipowermax-reverseproxy:v2.11.0

Reply:
This are versions in test file. This will be only used for UI unit tests. Even if do not change the version it should not create any issues.